packages/crud/test/crud-buttons.test.js

```diff
@@ -17,17 +17,46 @@ describe('crud buttons', () => {
         if (mode === 'default') {
           crud = fixtureSync('<vaadin-crud style="width: 300px;"></vaadin-crud>');
         } else {
-          crud = fixtureSync(
-            `<vaadin-crud style="width: 300px;">
+          crud = fixtureSync(`
+            <vaadin-crud style="width: 300px;">
               <vaadin-button slot="save-button"></vaadin-button>
               <vaadin-button slot="cancel-button"></vaadin-button>
               <vaadin-button slot="delete-button"></vaadin-button>
-            </vaadin-crud>`
-          );
+            </vaadin-crud>
+          `);
         }
         crud.items = [{ foo: 'bar' }];
         await nextRender(crud._grid);
-        [btnSave, btnCancel, btnDelete] = crud.querySelectorAll('vaadin-button');
+        [saveButton, cancelButton, deleteButton] = crud.querySelectorAll('vaadin-button');
```

Comment:
**note:** Decided to rename the buttons to align their naming with those used in the actual implementation. It makes it easier to find them when searching across the project.